antora-ui-camel/src/css/doc.css

```diff
@@ -149,6 +149,16 @@
 .doc a {
   color: var(--link-font-color);
   word-break: break-word;
+  text-decoration: underline;
+  text-underline-position: under;
```

Comment:
This bit looks a bit weird to me.

```suggestion
```

Reply:
You require this particular line so that there is a minimum distance between the underline and the text.

Reply:
Right, so this is with `text-underline-position: under`:

![Screenshot_2020-08-11 Home - Apache Camel](https://user-images.githubusercontent.com/1306050/89947841-5e2b0600-dc25-11ea-87eb-0ee28caca42a.png)

and without:

![Screenshot_2020-08-11 Home - Apache Camel(1)](https://user-images.githubusercontent.com/1306050/89947917-7d299800-dc25-11ea-805b-4bacfc13cc4b.png)

I think what you're reacting to is that the underline is quite noticeable, perhaps adding to make it lighter:

```suggestion
  text-decoration-thickness: 1px;
  text-decoration-style: dotted;
```

Would help, this is how it renders in Firefox:

![Screenshot_2020-08-11 Home - Apache Camel(3)](https://user-images.githubusercontent.com/1306050/89948444-70f20a80-dc26-11ea-9a57-b84bd5659c42.png)

Reply:
I am not sure about to dotted style. It would look quite odd. I will look into the thickness.

Reply:
Thickness feature isn't present for chrome browser and the dotted style looks inconsistent. Either we just not include under or we include it. It's best to keep it simple. Or I could use the `border-bottom` property.